apps/wallet/src/utils/app.utils.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, expect, it } from 'vitest';
+import { BlockchainStandard, BlockchainType } from '~/types/chain.types';
+import {
+  isCaseInsensitiveBlockchainAddress,
+  maybeTransformBlockchainAddress,
+} from '~/utils/app.utils';
+
+describe('BlockchainAddress', () => {
+  it('ICP Ledger account identifiers are case insensitive', () => {
+    expect(
+      isCaseInsensitiveBlockchainAddress(
+        BlockchainType.InternetComputer,
+        BlockchainStandard.Native,
+      ),
+    ).toBeTruthy();
```

Comment:
I think `toBe(true)` would be more appropriate here since `toBeTruthy()` will succeed for all sorts of things.

Some tests for the negative case in this and the other test here would be nice too, but I don't think it's a big deal since the logic is simple.

Reply:
Updated to `toBe(true)`. Since the ICP standard is the only possible option so far there's no negative case, we can add those once we support other types of addresses.